assets/css/calypsoify.css

```diff
@@ -316,6 +316,105 @@ table.widefat {
 	color: #2E4453;
 }
 
+/* Forms */
+
+#post-body-content {
+	background-color: #fff;
+	box-shadow: 0 0 0 1px rgba(200,215,225,0.5), 0 1px 2px #e9eff3;
```

Comment:
Can we simplify this to a `border: 1px solid #hex` here?  It appears to be a single solid border around its edge, but I may be missing some details here.